packages/rs-drive-abci/src/execution/engine/initialization/init_chain/v0/mod.rs

```diff
@@ -79,6 +79,24 @@ where
 
         state_guard.set_genesis_block_info(Some(genesis_block_info));
 
+        // Store initial protocol version
+        let mut batch_operations = vec![];
+
+        self.drive.set_current_protocol_version_operations(
+            state_guard.current_protocol_version_in_consensus(),
+            Some(transaction),
+            &mut batch_operations,
+            &platform_version.drive,
+        )?;
+
+        self.drive.apply_batch_low_level_drive_operations(
+            None,
+            Some(transaction),
+            batch_operations,
+            &mut vec![],
+            &platform_version.drive,
+        )?;
```

Comment:
Operations are for a batch of things you want to do, for every operations call there should also be a direct call. So we should add a store_current_protocol_version(, also we should name it store instead of set, as it's not a setter and can be confusing for some.